src/pods/canvas/components/canvas-accessible/components/field-accessible.component.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { FieldVm } from '@/core/providers';
+import { FieldList } from './field-list-accessible.component';
+
+interface Props {
+  field: FieldVm;
+  listName: string;
+}
+
+export const Field: React.FC<Props> = props => {
+  const { field, listName } = props;
+
+  const isNN = (NN?: boolean) => {
+    if (NN) return <span>NN</span>;
+  };
+
+  const isChildren = (name: string, children?: FieldVm[]) => {
+    if (children)
+      return <FieldList fieldList={children} listName={`${name} nested`} />;
+  };
+
```

Comment:
Change the names of isNN and isChildren to something more descriptive, e.g. handleNN or renderNNElement.